test/utils/check-answers.js

```diff
@@ -474,6 +474,7 @@ module.exports = {
   expectedEndemicsFollowUpDairy,
   expectedEndemicsFollowUpPigs,
   expectedEndemicsFollowUpSheep,
+  sheepTestResults,
```

Comment:
Why do we need to export this? It isn't used anywhere as far as I can see

Reply:
It's been used in check-answers.test.js

Reply:
Do we need to export it just to test it, is it being flagged as an uncovered line?

Reply:
Yes, we need it to cover line 286 to 290

Reply:
Is it used anywhere, if it isn't being covered by tests then that implies that it isn't actually being used. Or can we test wherever it is used instead?